src/richie/apps/courses/models/course.py

```diff
@@ -318,26 +318,43 @@ def create_permissions_for_organization(self, organization):
             ),
         )
 
-    def get_persons(self, language=None):
+    def get_page_extensions(self, extension_model, plugin_model, language=None):
```

Comment:
What do you think to move this method within `BasePageExtension` next to `get_related_page_extensions` as nothing used inside is related to the `Course` class and this method could be useful for other models inherit from `BasePageExtension`?